Weather/Weather/Scenes/Search/SearchLocationViewController.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  SearchLocationViewController.swift
+//  Weather
+//
+//  Created by minh duc on 2/19/19.
+//  Copyright © 2019 minhduc. All rights reserved.
+//
+
+import UIKit
+
+class SearchLocationViewController: BaseTableViewController {
+    @IBOutlet private weak var searchBar: UISearchBar!
+    
+    private let searchRepo = SearchRepository()
+    private var predictions = [Prediction]()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    override func configView() {
+        searchBar.do {
+            $0.delegate = self
+            $0.becomeFirstResponder()
+        }
+        UITextField.appearance(whenContainedInInstancesOf: [UISearchBar.self]).textColor = UIColor.white
+    }
+    
+    override func configTableView() {
+        super.configTableView()
+        baseTableView.do {
+            $0.register(cellType: SearchLocationCell.self)
+        }
+    }
+    
+    @IBAction func dismissViewController(_ sender: UIButton) {
+        searchBar.resignFirstResponder()
+        dismiss(animated: true, completion: nil)
+    }
+}
+//MARK: - Override
```

Comment:
```suggestion
// MARK: - Override
```
Comment sai coding convention